src/lib/containers/CardContainer.tsx

```diff
@@ -111,8 +111,7 @@ export const CardContainer = (props: CardContainerProps) => {
       )
     }
     const listIds = data.queryResult!.queryResults.rows.map(
-      el =>
-        el.values.filter((id): id is string => id !== null)[userIdColumnIndex],
+      el => el.values[userIdColumnIndex],
```

Comment:
This bad filter caused the issue. Null data would be dropped and the index would be wrong! We'd then send this data over to Synapse assuming it's a list user IDs and it turns out to be a list of arbitrary strings.

We don't actually want to filter anything; nulls are OK and represent fake profiles.